consensus/safety-rules/src/safety_rules.rs

```diff
@@ -136,10 +136,8 @@ impl<T: Payload> SafetyRules<T> {
             Ok(())
         }
     }
-}
 
-impl<T: Payload> TSafetyRules<T> for SafetyRules<T> {
-    fn consensus_state(&mut self) -> Result<ConsensusState, Error> {
```

Comment:
why? we implemented the interface, there's no point in not exposing this via this trait, is there?